### ISSUE TITLE
5.9.2 shows libraries which are not used

### ISSUE THREAD

**Issue by rubengees** [mikepenz/AboutLibraries#304]
As the title says, the latest update shows libraries which are not used or declared in the app.

Namely for me: `ACRA`, `Android Holo ColorPicker`, `Android-Job`, `Avatar ImageView` and `Earl`.
My load line is quite simple:

```java
new LibsBuilder().withAutoDetect(false)
        .withAboutVersionShown(false)
        .withAboutIconShown(false)
        .withAboutDescription("Description")
        .withLibraries(LIBRARIES) // Defined as a static Array
        .withActivityStyle(Libs.ActivityStyle.LIGHT_DARK_TOOLBAR)
        .withFields(R.string.class.getFields())
        .withActivityTitle("Title")
        .start(this);
```

I have setup a sample showcasing this [here](https://github.com/rubengees/AboutLibrariesBug).
Thanks in advance for looking into this!

**Comment by Rainer-Lang** [comment]
I'll look into it...

**Comment by mikepenz** [comment]
@Rainer-Lang well they miss the "_int" :P

**Comment by Rainer-Lang** [comment]
@mikepenz I know I'm fixing it. :)

**Comment by mikepenz** [comment]
fixed in v5.9.3

**Comment by Rainer-Lang** [comment]
@rubengees Thanks for this issue.
